trillian-logserver.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: trillian-logserver
+  version: 1.5.2
+  epoch: 0
+  description: Merkle tree implementation used in Sigstore
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - go
```

Comment:
Not blocking, but this isn't needed since using `go/install` brings this dep in by itself. Specifying it again doesn't hurt it's just not needed.